forwarder.go

```diff
@@ -219,15 +246,16 @@ func init() {
 		MaxIdleConnsPerHost: workers,
 	}
 	client = &http.Client{Transport: transport}
+	timestampRegex, _ = regexp.Compile("([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])[Tt]([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(.[0-9]+)?(([Zz])|([+|-]([01][0-9]|2[0-3]):[0-5][0-9]))")
 
-	flag.StringVar(&fwdURL, "url", "", "The url to forward to")
+	flag.StringVar(&fwdURL, "url", "http://localhost:8080", "The url to forward to")
```

Comment:
It's useful when you do a local testing and you don't want to post messages to splunk. I removed it.